dp/cloud/go/services/dp/obsidian/models/validation_test.go

```diff
@@ -83,6 +83,14 @@ func TestMutableCbsd_Validate(t *testing.T) {
 		name:          "Should validate max ibw mhz",
 		data:          b.NewMutableCbsdModelPayloadBuilder().WithMaxIbwMhz(0).Payload,
 		expectedError: "max_ibw_mhz in body is required",
+	}, {
+		name:          "Should validate max ibw mhz too high",
+		data:          b.NewMutableCbsdModelPayloadBuilder().WithMaxIbwMhz(151).Payload,
+		expectedError: "max_ibw_mhz in body should be less than or equal to 150",
+	}, {
+		name:          "Should validate max ibw mhz not multiple of 5",
+		data:          b.NewMutableCbsdModelPayloadBuilder().WithMaxIbwMhz(7).Payload,
+		expectedError: "max_ibw_mhz in body should be a multiple of 5",
```

Comment:
Added them in a separate test, since original `TestMutableCbsd_Validate` covers only swagger generated `Validate` func and would not cover `ValidateModel` func.